test/unit/getTableName-test.js

```diff
@@ -1,19 +0,0 @@
-require('require-self-ref')
-
```

Comment:
I put in the PR description: "The deletions were a few accidental duplicate tests."